src/main/java/com/google/collegeplanner/data/Meeting.java

```diff
@@ -205,8 +218,32 @@ public String getDaysAsString() {
     if (days.contains(DayOfWeek.FRIDAY)) {
       daysString += "F";
     }
-    // Do not need to check for Saturday or Sunday as the constructor throws an error
-    // if user tries to add those days into the list.
+    // Do not need to check for Saturday or Sunday as the constructor 
+    // throws an error if user tries to add those days into the list.
     return daysString;
   }
+
+  /**
+   * Converts total minutes after midnight
+   * @param time Total minutes after 00:00
+   */
+  private String timeInString(int time) {
+    String timeString = "";
+    String amPm = "am";
+    int hours = time / 60;
+    int minutes = time % 60;
+    // This formatter has a format() function that takes in an integer 
+    // and returns a string of the number represented in two digits. 
+    // Examples: 1 -> "01"      12 -> "12"
+    DecimalFormat formatter = new DecimalFormat("00");
+    if (hours > 12) {
+      hours -= 12;
+    } else if (hours == 0) {
+      hours = 12;
+    }
+    if (hours >= 12) {
+      amPm = "pm";
```

Comment:
If the time is 10 mins after midnight, will it be changed to "pm" instead?

Reply:
I messed up the logic slightly, will fix it